cli/addressbook/address.go

```diff
@@ -0,0 +1,118 @@
+package addressbook
+
+import (
+	"fmt"
+
+	"github.com/AlecAivazis/survey/v2"
+	ethCommon "github.com/ethereum/go-ethereum/common"
+	"github.com/mitchellh/mapstructure"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/helpers"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/types"
+	"github.com/spf13/cobra"
+)
+
+const (
+	// Kind is the account kind for the test accounts.
+	Kind = "address"
+
+	// ethAddressCfgKey is an optional key passed in the config files.
+	ethAddressCfgKey = "eth_address"
+)
+
+// ImportSource is a source of imported address account.
+type ImportSource struct {
+	Data string
+}
+
+type addressAccountConfig struct {
+	EthAddress string `mapstructure:"eth_address,omitempty"`
+}
+
+type AddressAccount struct {
+	address types.Address
+	cfg     *addressAccountConfig
+}
+
+func newAccount(address types.Address, cfg *addressAccountConfig) (*AddressAccount, error) {
+	return &AddressAccount{address: address, cfg: cfg}, nil
+}
+
+type AddressAccountFactory struct{}
+
+func (a *AddressAccountFactory) unmarshalConfig(raw map[string]interface{}) (*addressAccountConfig, error) {
+	if raw == nil {
+		// No eth_address defined.
+		return &addressAccountConfig{}, nil
+	}
+
+	var cfg addressAccountConfig
+	if err := mapstructure.Decode(raw, &cfg); err != nil {
+		return nil, err
+	}
+	return &cfg, nil
+}
+
+func (a *AddressAccountFactory) Create(name string, cfg map[string]interface{}) (*AddressAccount, error) {
+	return nil, fmt.Errorf("address: create not supported")
+}
+
+func (a *AddressAccountFactory) Load(_ string, rawCfg map[string]interface{}, rawAddr string) (*AddressAccount, error) {
+	cfg, err := a.unmarshalConfig(rawCfg)
+	if err != nil {
+		return nil, err
+	}
+
+	addr, _, err := helpers.ResolveEthOrOasisAddress(rawAddr)
+	if err != nil {
+		return nil, err
+	}
+
+	return newAccount(*addr, cfg)
+}
+
+func (a *AddressAccountFactory) Import(cfg map[string]interface{}, src *ImportSource) (*AddressAccount, error) {
```

Comment:
Is there a reason to have a more complicated import process? For accounts I get it, but is it really needed for addresses? Would just a simple add be enough?